static/src/javascripts/projects/commercial/modules/liveblog-adverts.js

```diff
@@ -15,12 +17,18 @@ let firstSlot;
 
 const startListening = () => {
 	// eslint-disable-next-line no-use-before-define
```

Comment:
This is a real nitpick and not something you introduced but i think we can remove the two occurrences of `// eslint-disable-next-line no-use-before-define`.

EDIT: And non-blocking of course.